packages/siop-oid4vp/lib/authorization-response/PresentationExchange.ts

```diff
@@ -324,7 +383,7 @@ export class PresentationExchange {
       throw new Error(SIOPErrors.NO_PRESENTATION_SUBMISSION)
     }
 
-    if (evaluationResults.areRequiredCredentialsPresent === Status.ERROR || evaluationResults.errors?.length || !evaluationResults.value) {
+    if (!evaluationResults.areRequiredCredentialsPresent || (evaluationResults.errors && evaluationResults.errors.length > 0) || !evaluationResults.value) {
```

Comment:
The first part of the expression all of a sudden does something very different from the previous version, given it is an enum

Reply:
weird.
I suppose I meant to only tighten up the second part.

Reply:
(fixed)